src/lib.rs

```diff
@@ -62,6 +62,65 @@
 //!
 //!     loop {}
 //! }
+//! ```
+//!
+//! ## Notes
+//! 
+//! On the bluepill (STM32F103) the pin seems to be pulled low by default causing the sensor
+//! to be confused when actually reading it for the first time. The same thing happens when
+//! the sensor is polled too quickly in succession. In this case you will get a `DhtError::Timeout`
+//!
+//! To avoid this, you can pull the pin high when initializing it and polling the sensor with an
+//! interval of at least 500ms.
```

Comment:
I think you can add this before the above `## Example` heading, worded in a non chip specific way